spiketoolkit/postprocessing/postprocessing_tools.py

```diff
@@ -11,7 +11,7 @@
 def get_unit_waveforms(recording, sorting, unit_ids=None, grouping_property=None, channel_ids=None,
                        ms_before=3., ms_after=3., dtype=None, max_spikes_per_unit=np.inf,
                        save_as_features=True, compute_property_from_recording=False, verbose=False,
-                       seed=0):
+                       seed=0, return_idxs=False):
```

Comment:
I think for performence reason both "return_idxs" and "waveforms" should be numpy arrays preallocated at the good lenght.

Reply:
waveforms and indexes can have differet shapes. Even if max_spikes_per_unit=300, if a spike train has less than 300 spikes than we can't pre-allocate memory..I'm afraid we have tu use lists